app/react/Attachments/components/FileList.tsx

```diff
@@ -38,15 +39,29 @@ export class FileList extends Component<FileListProps> {
     );
   }
 
-  render() {
-    const { files } = this.props;
+  orderFilesByLanguage(files: FileType[], systemLanguage: string) {
+    const orderedFiles = [...files];
+    const fileIndex = orderedFiles.findIndex(file => {
+      const language = languageLib.get(file.language as string, 'ISO639_1');
+      return language === systemLanguage;
+    });
+    if (fileIndex > -1) {
+      const temp = orderedFiles[fileIndex];
+      orderedFiles[fileIndex] = orderedFiles[0];
+      orderedFiles[0] = temp;
+    }
+    return orderedFiles;
```

Comment:
@grafitto if i comment this code, the tests are still green, the test sets up an scenario where the first file is already the system language. this is something that we want to probably test, but we need to test also when this is not the case.

Reply:
Done

Reply:
@daneryl